CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest
+- Changed default resolution of Camera2VideoSourceImpl to 960*540
+- Added param to Camera2VideoSourceImpl constructor to enable\disable autofocus
+- Added support of reverse landscape orientation
+- Removed deprecated code for 0.12.0
```

Comment:
I'd listed specific deprecated code that was removed

Reply:
Added, @kiryldz please check removed deprecated code as it is related to this PR 
https://github.com/mapbox/mapbox-vision/pull/2584